org/views.py

```diff
@@ -54,46 +54,3 @@ def post(self,request):
             data = serializer.errors
             return Response(data, status.HTTP_400_BAD_REQUEST)
 
-
-@swagger_auto_schema(
-    operation_id="add_volunteer",
-    operation_description="When an authenticated user hits this API it gets added to the volunteer group",
-    method='get',
-    responses={
-        '201': set_example(responses.add_volunteer_201),
-        '400': set_example(responses.org_not_present_400),
-        '401': set_example(responses.user_unauthorized_401),
-        '409': set_example(responses.user_already_present_409)
-    }
-)
-@api_view(['get'])
-@permission_classes([IsAuthenticated])
-def AddVolunteer(request,org_id):
-    org_count = Org.objects.filter(pk=org_id).count()
-    
-    if org_count>0:
-        org = Org.objects.get(pk=org_id)
-        member_present = Member.objects.filter(
-            user = request.user,
-            org = org 
-        ).count()
-        if member_present>0:
-            return Response({"message":"Already a member of the organization"},status.HTTP_409_CONFLICT)
-        else:
-            volunteer_permission_set = PermissionSet.objects.get(
-                name='Volunteer',
-                org=org,
-            )
-            volunteer_group = Group.objects.get(
-                name='Volunteer',
-                org=org,
-            )
-            member = Member.objects.create(
-                user = request.user,
-                org = org,
-                group = volunteer_group,
-                permissions = volunteer_permission_set 
-            )
-            return Response({"message":"You are added as a volunteer"},status.HTTP_201_CREATED)
-    else:
```

Comment:
@KunalAnkur You removed ```AddVolunteer``` view from ```org/views.py``` which is failing the test, please pull the existing code by ```git pull upstream dev``` and push your changes after editing that.

Reply:
And please locally test the project by ```python manage.py test``` , and only push the code if the checks passes locally.

Reply:
okay i got it